fixtures/refs.yaml

```diff
@@ -15,3 +15,10 @@ definitions:
         additionalProperties:
           type: string
         x-proto-tag: 12
+      test_map_array:
```

Comment:
Can we add one more test case that covers the generic `ListValue` type we've also added?